src/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -52,25 +53,35 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
   const isPrivateSale = source === "private_sale"
   const isPartnerOffer = source === "partner_offer"
 
-  const priceLabel = ((mode === "OFFER") || isPartnerOffer) ? "List price" : "Price"
+  const priceLabel = mode === "OFFER" || isPartnerOffer ? "List price" : "Price"
 
   return (
     <StackableBorderBox flexDirection="row" {...others}>
       <Box height="auto">
-        {imageURL && <Image src={imageURL} alt={title!} width="55px" mr={1} />}
+        {imageURL && (
+          <Link href={`/artwork/${artwork?.slug}`} target="_blank">
```

Comment:
For private offers, we may not have a public artwork page for the artwork that is being purchased. If the order `mode` is available, it may be worth checking before adding the href.

Reply:
Great point! I'll check